### PR TITLE
Update Open edX course author docs per CCX dashboard changes

### PR DESCRIPTION
**Pull request by srpearce**
## [DOC-3881](https://openedx.atlassian.net/browse/DOC-3881)

Update Open edX custom course (CCX) information to reflect changes to the Enrollment tab on the dashboard. More information in [OSPR-2074](https://openedx.atlassian.net/browse/OSPR-2074).

### Date Needed (optional)

9 Feb 2018

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 

- [ ] Subject matter expert: @tomaszgy
- [x] Subject matter expert: @dsjen 
- [ ] Doc team review (copy edit?): @edx/doc
- [ ] Product review: @sstack22 
- [ ] Partner support: 
- [ ] PM review: 

### Testing

- [ ] Ran ./run_tests.sh without warnings or errors

### Post-review

- [ ] Add a comment with the description of this change or link this PR to the next release notes task.
- [ ] Squash commits

### CODE REVIEW

File: en_us/open_edx_course_authors/source/set_up_course/studio_add_course_information/custom_courses.rst [hunk 1]
```diff
@@ -16,16 +16,16 @@ Overview
...
 A CCX can contain only content from an existing course; you cannot author new
-content in a CCX. 
+content in a CCX.
 
 **************************
 Custom Courses and Roles
 **************************
 
-There are three roles involved in creating and running a CCX. 
+There are three roles involved in creating and running a CCX.
```
Comment (debchatigny) [inline en_us/open_edx_course_authors/source/set_up_course/studio_add_course_information/custom_courses.rst:28]: It's not part of this PR, but part of this file (so I don't see an option to add the comment there): Line 53: Can you please change **Click** to **Select**? Thanks.

**Review by dsjen — APPROVED** [review]
LGTM!

**Comment by debchatigny** [comment]
Other than the small, non-PR-related item on line 53, all good! 👍

**Comment by srpearce** [comment]
From OSPR author on [platform PR 17160](https://github.com/edx/edx-platform/pull/17160#issuecomment-364192645):

In theory:
* you can revoke access from a user that has an active edX account and is enrolled in a course
* you can unenroll the user mentioned above but also someone who has been invited to a course but doesn't have an active edX account (she/he needs to activate his/her user account through the link sent in an invitation e-mail)

Also, Student List Management displays only enrolled students with active account.
